File: output/x_ai_grok_code_fast_1/zero/cpp/Prêmio do Milhão.cpp

#include <iostream>

int main() {
    int N;
    std::cin >> N;
    long long soma = 0;
    int dias = 0;
    for (int i = 0; i < N; ++i) {
        int A;
        std::cin >> A;
        soma += A;
        dias++;
        if (soma >= 1000000) {
            break;
        }
    }
    std::cout << dias << std::endl;
    return 0;
}

File: output/csbc_semish_2026/x_ai_grok_code_fast_1/few/python/Prêmio do Milhão.py
import sys

data = sys.stdin.read().split()
N = int(data[0])
acessos = [int(x) for x in data[1:]]
soma = 0
dias = 0
for a in acessos:
    soma += a
    dias += 1
    if soma >= 1000000:
        break
print(dias)